### DISCUSSION TITLE
关于 snlua 中 sn 的缩写

### DISCUSSION THREAD

**Discussion by ylme** [cloudwu/skynet#1620] [General]
之前一直在猜 snlua 中 sn 是什么的缩写，一直没用想清楚，直到无意间读到 [Skynet 开源](https://blog.codingnow.com/2012/08/skynet.html) 下面这段：

> lua 服务是另一项基础设施，但不属于核心部分。如果你喜欢，也可以用 python 等其它动态语言替换掉。
这里叫 snlua ，以和系统内的 lua 区分。

恍然大悟，原来 sn 是 skynet 的缩写，哈哈哈。你是不是也是才知道呢

**Comment by terry8210** [comment]
用sk不是更好吗？

**Reply by ylme** [reply]
缩写的事情没有绝对吧，取决于作者

**Comment by CoolDesert** [comment]
关注久的人基本知道，不然你再猜猜 snax 是啥缩写：）
